settings.gradle.kts

```diff
@@ -39,7 +39,7 @@ gitRepositories {
     include("hedera-protobufs") {
         uri.set("https://github.com/hashgraph/hedera-protobufs.git")
         // choose tag or branch of HAPI you would like to test with
-        tag.set("main")
+        tag.set("v0.34.0")
```

Comment:
Some comments would be useful, to say that the version number needs to be set both here and below.

Reply:
Done!